### PR TITLE
Issue 3022: Content Update: Segment-containers.md

### LINKED ISSUE #3022: Document review: segment-containers.md
**Problem description**
1. Format fixing
2. hyperlinks checking

### PR DESCRIPTION
**Pull request by Aparnarr**
Signed-off-by: Aparna <Aparna.Rr@emc.com>

**Change log description**  
1. Minor content edits
 2. Last few sections ends up with **This needs further review/discussion.**
   @andreipaduroiu , please address this !

**Purpose of the change**  
 Fixes #3022

### CODE REVIEW

File: documentation/src/docs/segment-containers.md [hunk 1]
```diff
@@ -12,25 +12,25 @@ You may obtain a copy of the License at
...
 
 # Ownership Change Notification
-Every pravega host has a long running Segment Manager Service. This constantly polls/watches the HostStore for any changes to the container ownership. On detecting any ownership changes for itself (identified by the host key in the Map) the Segment Manager triggers addition and removal of containers accordingly.
+Every Pravega host has a long running Segment Manager Service. This constantly polls/watches the HostStore for any changes to the container ownership. On detecting any ownership changes for itself (identified by the host key in the Map) the Segment Manager triggers addition and removal of containers accordingly.
 
 ## Ensuring a container is stopped on one host before it is started on another host
 We currently plan to use conservative timeouts on the Pravega host to ensure a container is not started before it is stopped on another node. This needs further review/discussion.
```
Comment (Aparnarr) [inline documentation/src/docs/segment-containers.md]: @andreipaduroiu @fpj 
Any updates on this??

File: documentation/src/docs/segment-containers.md [hunk 2]
```diff
@@ -12,25 +12,25 @@ You may obtain a copy of the License at
...
+We use Zookeeper to ensure only one Pravega Controller is monitoring the cluster to avoid multiple simultaneous Writers to the HostStore. This will avoid race conditions and allow the state to converge faster.
 
 ## Rebalance Frequency
-Rebalance of the container ownership happens when either a pravega host is added or removed from the cluster. Since this is a costly operation we need to guard against doing this very often. Currently we ensure that a configured minimum time interval is maintained between any 2 rebalance operations. In the worst case this will proportionally increase the time it takes to perform ownership change in the cluster.
+Rebalance of the Container ownership happens when either a Pravega Host is added or removed from the cluster. Since, this is an expensive operation and not to be repeated very often. Currently we ensure that a configured minimum time interval is maintained between any two rebalance operations. In the worst case, this will proportionally increase the time it takes to perform ownership change in the cluster.
```
Comment (andreipaduroiu) [inline documentation/src/docs/segment-containers.md]: remove the comma.

Comment (andreipaduroiu) [reply documentation/src/docs/segment-containers.md]: Reword this sentence. It doesn't make any sense in the current form.

File: documentation/src/docs/segment-containers.md (hunk 1 above)
Comment (andreipaduroiu) [reply documentation/src/docs/segment-containers.md]: no

Comment (Aparnarr) [reply documentation/src/docs/segment-containers.md]: @andrei Cani remove the line "This needs further review/discussion".??

Comment (andreipaduroiu) [reply documentation/src/docs/segment-containers.md]: yes, remove that. Actually we should probably remove this whole paragraph

**Comment by Aparnarr** [comment]
@andreipaduroiu 
Please check the last section, should that also be removed??

**Comment by andreipaduroiu** [comment]
@Aparnarr Yes, you can remove that whole section that says "this needs further review"

**Pull request merged**